fv3config/_asset_list.py

```diff
@@ -231,6 +165,8 @@ def write_asset(asset, target_directory):
         logger.debug(f"Writing asset bytes to {target_path}.")
         with open(target_path, "wb") as f:
             f.write(asset["bytes"])
+    elif "directory" in asset:
+        return os.makedirs(target_path, exist_ok=True)
     else:
         raise ConfigError(
             "Cannot write asset. Asset must have either a `copy_method` or `bytes` key."
```

Comment:
Maybe update this message to something like: "Cannot write asset.  Asset must be a directory or have either a \`copy_method\` or \`bytes\` key."

Reply:
i changed to making `directory` a `copy_method`.